src/main/kotlin/probation/CRCSystem.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.hmpps.architecture.probation
+
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+import uk.gov.justice.hmpps.architecture.shared.Tags
+
+class CRCSystem private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "CRC software systems",
+        "Various systems used to plan, schedule, manage interventions and bookings"
+      ).apply {
+        Tags.PROVIDER.addTo(this)
+        ProblemArea.GETTING_THE_RIGHT_REHABILITATION.addTo(this)
+      }
+    }
```

Comment:
Should we separate out the CRC systems, and group them somehow (maybe with a tag?)

Reply:
Do you mean the different types?

Reply:
🤷 are there many (sorry, I don't know this space well)? If there's a lot it would introduce noise to the diagram. Maybe we could visually highlight that there's many with a stack of boxes or something similar?

Reply:
Hmm, good question. My feeling is that a reminder that external systems exist is currently enough narrative for our space. The important is that we know these exist and that there are ones that integrate with us and most that don't.

Thank you for the question! I'll update to reflect the integrated/standalone external systems. ⌛

Reply:
Done in c7aa37d76fe9506314fa4287760b71233c414f2b